package.json

```diff
@@ -4,7 +4,7 @@
   "license": "UNLICENSED",
   "type": "module",
   "scripts": {
-    "install_all": "(cd frontend && yarn install) && (cd backend && yarn install) && (cd shared && yarn install)",
+    "install_all": "yarn install && (cd frontend && yarn install) && (cd backend && yarn install) && (cd shared && yarn install)",
```

Comment:
I think it would be cleaner since the standard is to run `yarn install` (people usually run it without reading the doc)

Reply:
@sandoche Great idea! I'll update that next time I'm working on the `package.json`.

Reply:
@sandoche What do you think about https://yarnpkg.com/advanced/lifecycle-scripts#a-note-about-postinstall ? It's making me hesitate.

Reply:
Hmm yeah I am not sure I understand what they say. Since we will run `yarn install` the caching will still happen no?

Here are some people are using this and no one mentionned this lifecycle problem: https://stackoverflow.com/questions/31773546/the-best-way-to-run-npm-install-for-nested-folders

Reply:
@sandoche  Right, but I'm guessing it's because of differences between npm and yarn. We could switch to npm, but Pierre and I checked yesterday at https://www.google.com/search?q=yarn+vs+npm+2022 and saw that yarn is supposedly still faster and more secure than npm (but uses more disk space).

Reply:
Actually if we want to improve the speed we should even move to pnpm

Reply:
And it also looks like they have special features for mono repo projects